google-cloud-spanner-hibernate-testing/src/test/java/org/hibernate/userguide/hql/HQLTest.java

```diff
@@ -1469,24 +1432,6 @@ public void test_hql_collection_expressions_example_4() {
 		});
 	}
 
-	@Test
-	public void test_hql_collection_expressions_example_5() {
-		doInJPA( this::entityManagerFactory, entityManager -> {
-			Call call = entityManager.createQuery( "select c from Call c", Call.class).getResultList().get( 0 );
-			Phone phone = call.getPhone();
-			//tag::hql-collection-expressions-example[]
-
-			List<Person> persons = entityManager.createQuery(
-				"select p " +
-				"from Person p " +
-				"where :phone = some elements ( p.phones )", Person.class )
```

Comment:
this  `= som elements ( ....` becomes ` = ANY (... ` ?

Reply:
No, turns out `SOME` is [its own keyword](https://www.geeksforgeeks.org/sql-some/). Spanner does not yet support "SOME".